app/models/tournament.rb

```diff
@@ -52,7 +52,7 @@ def runner_counts
   end
 
   def generate_slug
-    self.slug = rand(36**4).to_s(36).upcase
+    self.slug = rand(Integer(36 ** 4)).to_s(36).upcase
```

Comment:
why is this change in this PR?

Reply:
It's extra effort to raise a separate PR and I have a limited amount of time.

Reply:
seems that way, but a change like this doesn't need a review and you could probably handle it in 2m tops.  Unrelated changes in PRs that are impactful tend to be harder to track down.  This isn't a big deal here, but please try to keep the changes focused on the problem for the PR.